migrations/2_dev_dependencies.js

```diff
@@ -1,7 +1,14 @@
 /*eslint no-undef: "off"*/
 
 const Dependencies = artifacts.require("./DevDependencies.sol")
+const BiMap = artifacts.require("BiMap.sol")
```

Comment:
Can we make sure this only happens on local (ganache) deploys somehow. Wouldn't want to deploy the wrapper on mainnet.